web/templates/settings/admin_invites_list.hbs

```diff
@@ -32,13 +32,34 @@
         <span>{{invited_as_text}}</span>
     </td>
     <td class="actions">
-        {{#unless is_multiuse}}
-        <button class="button rounded small resend button-warning" {{#if disable_buttons}}disabled="disabled"{{/if}} data-invite-id="{{id}}">
-            {{t "Resend" }}
-        </button>
-        {{/unless}}
-        <button class="button rounded small revoke button-danger" {{#if disable_buttons}}disabled="disabled"{{/if}} data-invite-id="{{id}}" data-is-multiuse="{{is_multiuse}}">
-            {{t "Revoke" }}
+        {{#if is_multiuse}}
+            <button
+              class="button rounded small tippy-zulip-delayed-tooltip edit-invite button-warning"
+              {{#if disable_buttons}}disabled="disabled"{{/if}}
+              data-invite-id="{{id}}"
+              data-is-multiuse="{{is_multiuse}}"
```

Comment:
Why `data-is-multiuse` needed here. We currently allow editing only multiuse invites.